src/components/views/elements/PersistedElement.js

```diff
@@ -106,9 +105,15 @@ export default class PersistedElement extends React.Component {
             {this.props.children}
         </div>;
 
-        ReactDOM.render(content, getOrCreateContainer());
+        ReactDOM.render(content, getOrCreateContainer('mx_persistedElement_'+this.props.persistKey));
 
         return <div ref={this.collectChildContainer}></div>;
     }
 }
 
+PersistedElement.propTypes = {
```

Comment:
why not the inline
```
static propTypes = {
...
};
```
syntax which it seems most of our ES6 classes use?

Reply:
(more readable to have it at the top of a class rather than not in it and completely below it IMO)

Reply:
hmm, I thought we were mostly doing it this way. This probably is more legible though.

Reply:
`static propTypes`: 26 instances
`.propTypes = {`: 40 instances

so yes you're right, but I believe most of them are from before people realised babel lets us use define inline static variables and is more readable. Feel free to ignore though as you're right with mostly the old way